docs_src/api/core/Ch-APICoreMetadata.md

```diff
@@ -1,21 +1,14 @@
 # Core Metadata
 
-## Architecture Reference
+## 
```

Comment:
Missing the section title?

Reply:
Nope, missed removal of that header... ;-)